spec/helpers/user_helper_spec.rb

```diff
@@ -6,9 +6,9 @@
     let(:name) { "Alice Bloggs" }
     let(:user) { create(:user, name: name) }
 
-    it "returns just the name when there is no profile URL" do
+    it "returns email link when there is no profile URL" do
       user.create_identity(provider: :facebook)
-      expect(helper.user_profile_link(user)).to eq name
+      expect(helper.user_profile_link(user)).to eq "<a href=\"mailto://alice.bloggs@example.com\">Alice Bloggs</a>"
```

Comment:
This is a change in behaviour that I think we want @jpallen ?

Reply:
Hmm, yes, partly. I hadn't seen the `user_profile_link` helper. I think we can be a little cleverer there. This is good for now though, thanks!